tests/basic/storage_pool_test.go

```diff
@@ -2053,3 +2209,1705 @@ var _ = Describe("{VolUpdateAddDrive}", func() {
+				expectedSize := poolToBeResized.TotalSize * 2 / units.GiB
+				resizedPoolsMap[poolToBeResized.Uuid] = expectedSize
+
+				log.FailOnError(err, "Failed to check if Journal enabled")
+
+				log.InfoD("Current Size of the pool %s is %d", selPool.Uuid, poolToBeResized.TotalSize/units.GiB)
+				err = Inst().V.ExpandPool(selPool.Uuid, api.SdkStoragePool_RESIZE_TYPE_ADD_DISK, expectedSize)
+				dash.VerifyFatal(err, nil, "Pool expansion init successful?")
+			}
+
+			isjournal, err := isJournalEnabled()
+			log.FailOnError(err, "Failed to check if Journal enabled")
+			for selPoolID, expectedPoolSize := range resizedPoolsMap {
+				resizeErr := waitForPoolToBeResized(expectedPoolSize, selPoolID, isjournal)
+				dash.VerifyFatal(resizeErr, nil, fmt.Sprintf("Verify pool %s on expansion using add-disk", selPoolID))
+			}
+
+		})
+		opts := make(map[string]bool)
+		opts[scheduler.OptionsWaitForResourceLeakCleanup] = true
+		ValidateAndDestroy(contexts, opts)
+
+	})
+	JustAfterEach(func() {
+		defer EndTorpedoTest()
+		AfterEachTest(contexts, testrailID, runID)
+	})
+})
+
+var _ = Describe("{ResizeWithJrnlAndMeta}", func() {
+	//1) Deploy px with cloud drive and journal enabled.
+	//2) Create a volume on that pool and write some data on the volume.
+	//3) Get the metadata node
+	//4) Expand the pool with journal device
+	var testrailID = 51289
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/51289
+	var runID int
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("ResizeWithJrnlAndMeta", "Initiate pool expansion using resize-disk for the pool the with journal and metadata devices", nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+	var contexts []*scheduler.Context
+
+	stepLog := "should get the metadata node and expand the pool by resize-disk"
+
+	It(stepLog, func() {
+		log.InfoD(stepLog)
+		journalStatus, err := isJournalEnabled()
+		log.FailOnError(err, "err getting journal status")
+		dash.VerifyFatal(journalStatus, true, "verify journal device is enabled")
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("rsizedrvmeta-%d", i))...)
+		}
+		ValidateApplications(contexts)
+
+		stNodes := node.GetMetadataNodes()
+
+		if len(stNodes) == 0 {
+			dash.VerifyFatal(len(stNodes) > 0, true, "Metadata nodes found?")
+		}
+		stNode, err := GetRandomNodeWithPoolIOs(stNodes)
+		log.FailOnError(err, "error identifying node to run test")
+		stNodePools := stNode.Pools
+
+		var selectedPool *api.StoragePool
+		for _, pool := range stNodePools {
+			if pool.ID == 0 {
+				selectedPool = pool
+				break
+			}
+		}
+
+		stepLog := "Initiate pool expansion drive and restart PX"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+
+			poolToBeResized, err := GetStoragePoolByUUID(selectedPool.Uuid)
+			log.FailOnError(err, "Failed to get pool using UUID ")
```

Comment:
Same here add the UUID that is used to get the StoragePool, would be good for debugging purposes
log.FailOnError(err, "Failed to get pool using UUID [%s]", selectedPool.Uuid)

Reply:
fixed